java-extensions/hadoop-ext/src/main/java/org/apache/hadoop/fs/FileSystem.java

```diff
@@ -3794,6 +3796,7 @@ static final class Cache {
 
         FileSystem get(URI uri, Configuration conf) throws IOException {
             Key key = new Key(uri, conf);
+            LOGGER.info(String.format("%s FileSystem.get. key = %s", HadoopExt.LOGGER_MESSAGE_PREFIX, key.toString()));
```

Comment:
`CloudConfString()` may contain the user's ak/sk, print credentials in the log is not safe for the user.

Reply:
removed it.